src/sql/src/plan/statement/ddl.rs

```diff
@@ -2061,11 +2062,15 @@ pub fn plan_create_materialized_view(
         &stmt.columns,
     )?;
     let column_names: Vec<ColumnName> = desc.iter_names().cloned().collect();
-    if !stmt.non_null_assertions.is_empty() {
+
+    let MaterializedViewOptionExtracted {
+        assert_not_null, ..
```

Comment:
```suggestion
        assert_not_null,
        seen: _,
```

Seems more future proof.

Reply:
Done.